WordPressAuthenticator/Authenticator/WordPressAuthenticatorStyles.swift

```diff
@@ -107,12 +116,14 @@ public struct WordPressAuthenticatorStyle {
                 viewControllerBackgroundColor: UIColor,
                 textFieldBackgroundColor: UIColor,
                 buttonViewBackgroundColor: UIColor? = nil,
+                buttonViewTopShadowImage: UIImage? = UIImage(named: "darkgrey-shadow"),
                 navBarImage: UIImage,
                 navBarBadgeColor: UIColor,
                 navBarBackgroundColor: UIColor,
                 navButtonTextColor: UIColor = .white,
                 prologueBackgroundColor: UIColor = WPStyleGuide.wordPressBlue(),
                 prologueTitleColor: UIColor = .white,
+                prologueTopContainerChildViewController: UIViewController? = nil,
```

Comment:
⚠️  I think we should consider using an [`@autoclosure`](https://docs.swift.org/swift-book/LanguageGuide/Closures.html#ID543) for this. It's probably best to delay the creation of the `UIViewController` until we really need it. Especially since it feels really odd to submit a `UIViewController` in a `class` with a `*Style` in the name. 😄

Reply:
TIL'ed about `@autoclosure`! Thanks for suggestion, I updated to use it here: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/373/commits/c0dc5df9bb38371c4c1239a75e1dfc5a26683cc4

Reply:
Awesome!